### DISCUSSION TITLE
CSV export from Analytics

### DISCUSSION THREAD

**Discussion by mranden** [woocommerce/woocommerce#35537] [WooCommerce HPOS] [answered]
Hey! :) 

I am having a hard time finding out, why WooCommerce ads a **'** infront of all my negative numbers. 

It only ads it, if the CSV is send by mail. 

Here I am showing what I meaning: https://imgur.com/a/XlDwGHK

**Comment by barryhughes** [comment, marked as answer]
Hi @mranden. Unless this is connected to the [WooCommerce HPOS](https://github.com/woocommerce/woocommerce/discussions/categories/woocommerce-hpos) (High-Performance Order Storage) project then it would be better to [ask this elsewhere](https://woocommerce.com/document/how-to-get-help/) or, if you are confident it is a bug, you can [post a bug report directly](https://github.com/woocommerce/woocommerce/issues/new?assignees=&labels=&template=1-bug-report.yml). Thanks!
